scripts/get_orphaned_functions.py

```diff
@@ -0,0 +1,243 @@
+import os
+import re
+import csv
+
+# Define Working Directory, i.e, '/pecan
+main_dir = os.getcwd()
+
+# Define a list of color codes for printing. THis proves to be useful for distinction during debugging
+color_codes = [
+    "\033[30m",  # BLACK-0
+    "\033[31m",  # RED-1
+    "\033[32m",  # GREEN-2
+    "\033[33m",  # YELLOW-3
+    "\033[34m",  # BLUE-4
+    "\033[35m",  # MAGENTA-5
+    "\033[36m",  # CYAN-6
+    "\033[37m",  # WHITE-7
+    "\033[90m",  # BRIGHT_BLACK-8
+    "\033[91m",  # BRIGHT_RED-9
+    "\033[92m",  # BRIGHT_GREEN-10
+    "\033[93m",  # BRIGHT_YELLOW-11
+    "\033[94m",  # BRIGHT_BLUE-12
+    "\033[95m",  # BRIGHT_MAGENTA-13
+    "\033[96m",  # BRIGHT_CYAN-14
+    "\033[97m",  # BRIGHT_WHITE-15
+]
+
+# Reset
+ENDC = "\033[0m"
+
+# Modules to ignore while checking for orphaned functions
+ignored_modules = ["data.hydrology", "DART"]
```

Comment:
Thanks for this tip -- I hadn't looked closely at `data.hydrology` in a long time. Now that I have, I think we don't need it and have opened #3292 to delete it entirely.